src/App.js

```diff
@@ -35,13 +40,16 @@ function App() {
     const url1 = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`;
     // const url2 = `https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}/videos`;
     setError('');
-    console.log('findMovie')
-
     try {
       const response1 = await fetch(url1);
-      const movieResponse = await response1.json();
-      setSelectedMovie(movieResponse.movie);
-     } catch (error) {
+      if (response1.ok) {
```

Comment:
To further enhance this, it would be beneficial to move all network requests into a separate file and call those functions from there.e.